utils/include/ti_stl_helpers.h

```diff
@@ -0,0 +1,66 @@
+/*
+ *  Copyright (C) 2021 Texas Instruments Incorporated - http://www.ti.com/
```

Comment:
@s-jain1 Is this copyright date correct?

Reply:
Updated